thorlcr/activities/thdiskbaseslave.cpp

```diff
@@ -433,14 +433,17 @@ void CDiskWriteSlaveActivityBase::close()
             outraw.clear();
         }
 
-        Owned<IFileIO> tmpFileIO;
+        if (outputIO)
```

Comment:
looking again, if outputIO is unset, it implies neither 'out' or 'outraw' can be set either.

It won't make any practical difference, but clearer if this conditional block includes lines 417-434 too.

Reply:
ok, I will move it there.  thx